web/modules/custom/portland/modules/portland_glossary/src/Controller/GlossaryLookupController.php

```diff
@@ -38,13 +38,11 @@ public function lookup(string $uuids) {
 
       // Prepare see_also array.
       $see_also = array_map(fn($ref_node) => [
-        'title' => $ref_node->label(),
```

Comment:
The removal of these `title` fields appears to break the display of the glossary popup on standard content pages, making it fallback to the generic "Glossary Term" title. I'm going to assume it was a mistake and revert this change:

<img width="534" height="436" alt="Image" src="https://github.com/user-attachments/assets/6a797e0a-2aaf-43cd-b1a7-120f1efa6083" />